docs/code-samples/tutorials-config/2.8.0.conf

```diff
@@ -2,34 +2,34 @@ https://github.com/digital-asset/daml-finance/releases/download/ContingentClaims
 https://github.com/digital-asset/daml-finance/releases/download/ContingentClaims.Lifecycle/2.0.0/contingent-claims-lifecycle-2.0.0.dar .lib/contingent-claims-lifecycle.dar
 https://github.com/digital-asset/daml-finance/releases/download/ContingentClaims.Valuation/0.2.1/contingent-claims-valuation-0.2.1.dar .lib/contingent-claims-valuation.dar
 https://github.com/digital-asset/daml-finance/releases/download/Daml.Finance.Account/2.99.0.20230927.1/daml-finance-account-2.99.0.20230927.1.dar .lib/daml-finance-account.dar
-https://github.com/digital-asset/daml-finance/releases/download/Daml.Finance.Claims/2.0.99.20230927.1/daml-finance-claims-2.0.99.20230927.1.dar .lib/daml-finance-claims.dar
-https://github.com/digital-asset/daml-finance/releases/download/Daml.Finance.Data/2.0.0.99.20230927.1/daml-finance-data-2.0.0.99.20230927.1.dar .lib/daml-finance-data.dar
+https://github.com/digital-asset/daml-finance/releases/download/Daml.Finance.Claims/2.1.0/daml-finance-claims-2.1.0.dar .lib/daml-finance-claims.dar
```

Comment:
just fyi, this will not work until the packages are released. I think we should stay on a snapshot version for now, given that we will probably put out another pre-release before the final release. wdyt?

Reply:
I will bump them back. Can still test it locally, and we can then decide if we want to make a snapshot release for the sole purpose of testing the quickstarter. Btw, let's consider code up a script for doing these changes at some point.

Reply:
To save some time, I bumped the date in the snapshot version to 20201030 for all packages. If we decide to do a snapshot release (for the quickstarter) there will be a few packages with the same content, but that shouldn't matter much.